rigex-front-end/src/components/WellsDropdown/WellsDropdown.js

```diff
@@ -0,0 +1,106 @@
+import React, { useState, useRef } from 'react';
+import {
+  ClickAwayListener,
+  Grow,
+  Paper,
+  Popper,
+  MenuItem,
+  MenuList,
+} from '@material-ui/core';
+import MoreHorizIcon from '@material-ui/icons/MoreHoriz';
+
+import './wellsDropdown.scss';
+
+const WellsDropdown = ({ wellId, isSoloRig, handleWellDelete }) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const anchorRef = useRef(null);
+
+  const handleDropdownClick = (clickType, itemClicked) => {
+    if (clickType === 'ellipses' && !isOpen) {
+      setIsOpen(!isOpen);
+    } else if (clickType === 'click away') {
+      setIsOpen(!isOpen);
+    } else if (itemClicked) {
+      switch (itemClicked) {
+        case 'delete':
+          handleWellDelete(wellId);
+          setIsOpen(!isOpen);
+          break;
+        case 'edit':
+          setIsOpen(!isOpen);
+          break;
+        case 'move':
+          setIsOpen(!isOpen);
+          break;
+        default:
+          setIsOpen(!isOpen);
+          break;
+      }
```

Comment:
Try using a fall-through for the switch cases that do the same thing. 

`case 'edit':
 case 'move':
 default:
`
I can't remember if default can actually go in there, but give it a try and see if it works like you're wanting.

Reply:
Obviously formatted correctly lol

Reply:
I thought about that, the problem was that every case-specific operation would run from the top down, not just the setIsOpen. I guess I could do another name==='case' check but that seemed to defeat the purpose of the switch lol

Reply:
Alright, that makes sense.